### DISCUSSION TITLE
We are moving forward with the Calculator program development.

### DISCUSSION THREAD

**Discussion by PranishKansakar** [PranishKansakar/Tafesa240154#2] [Announcements]
Hello everyone,

I hope this message finds you well. I wanted to provide a quick update on the progress of our program's **development**.

I'm pleased to announce that I have created a development branch that includes the main menu along with five buttons and image. 

In the coming days, we'll be discussing the assignment of tasks to each team member. The several key features that need to be implemented, include:

1. Mortgage calculator
2. Currency calculator
3. Trip calculator

Please keep an eye out for more news and be ready to join our upcoming talks about who will work on what tasks.

Let's keep going and make our program success,

Pranish Kansakar

**Comment by Omonotuaga** [comment]
Thank you for your update  Pranish, I will also post an update soon with the development of the trip calculator.

Please let me know if you need something to be done for the completion of the program.

Thank you very much,

Autonomo Amano
